runtime/debugger.c

```diff
@@ -66,10 +66,20 @@ CAMLexport void caml_debugger_cleanup_fork(void)
 #include <netdb.h>
 #else
 #define ATOM ATOM_WS
-#include <winsock.h>
+#include <winsock2.h>
+#include <mswsock.h>
 #undef ATOM
+#ifdef HAS_AFUNIX_H
+#include <afunix.h>
+#else
+struct sockaddr_un
+{
+    ADDRESS_FAMILY sun_family;     /* AF_UNIX */
+    char sun_path[108];            /* pathname */
+};
+#endif /* HAS_AFUNIX_H */
```

Comment:
This duplication is unfortunate - perhaps this would be better added to `runtime/caml/osdeps.h` (inside `CAML_INTERNALS`)?

Reply:
Sadly, this is difficult to do. `ADDRESS_FAMILY` is provided by `<winsock2.h>`, so we have to include it before. This header pulls a lot of definitions from Windows, and it creates a lot of naming conflicts. I've added a note explaining that.